cranelift/codegen/src/isa/aarch64/lower.rs

```diff
@@ -1272,7 +1311,15 @@ fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(ctx: &mut C, insn: IRInst) {
             //   x >> 56
             let ty = ty.unwrap();
             let rd = output_to_reg(ctx, outputs[0]);
-            let rn = input_to_reg(ctx, inputs[0], NarrowValueMode::None);
+            /* re-enable this when we do the right thing below:
```

Comment:
nit: either uncomment, or can you open an issue to fix this later + mention the issue number here, please?

Reply:
Yep, opened #1537 for this.